CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ and this project adheres to
   and underlying layers. Queue backlog length metrics are flushed periodically.
   They can be used to esimtate an average queue length by request by dividing
   its value by the number of requests served.
+- [#4492](https://github.com/firecracker-microvm/firecracker/pull/4492): Added
+  `cpu-template-helper fingerprint verify` command that functions as a
+  combination of the fingerprint dump and compare commands. This is useful when
+  ensuring that a fingerprint has not changed since the baseline was taken.
```

Comment:
I this command really necessary? At least for me having 2 separate commands, each doing their own small portion of the work seems like more intuitive and simple design. Now with `verify` you are combining them just for the sake of running 1 command instead of 2. Also looking at the implementation the existence of `--output` parameter is confusing. As I can see it is used to specify path to which current fingerprint will be written, but does anyone expect something called `verify` to output anything except some true/false statement? Maybe if you rename it to something like `DumpCompare` it would be more understandable, but still I don't see much benefit from this.

Reply:
not necessary, but nice-to have :)

Reply:
Rethinking what you pointed out and doubting myself, I came back to the "minimalistic" style of Firecracker. The new command may be just my prediction of customers' need and we can defer this task. I'll remove it for now.